_posts/2022-03-29-security-update.markdown

```diff
@@ -0,0 +1,22 @@
+---
+title: "Security Update"
```

Comment:
Add affected versions

Reply:
whoops, I've not done a great job here either based on past post titles

Better ones: https://hhvm.com/blog/2021/01/25/hhvm-4.94.html (though that's not a security update)